cars/v1/vanilla/config.ini

```diff
@@ -16,7 +16,7 @@ jdk.unbundled.release_url = https://artifacts.elastic.co/downloads/elasticsearch
 
 docker_image=docker.elastic.co/elasticsearch/elasticsearch
 # major version of the JDK that is used to build Elasticsearch
-build.jdk = 12
+build.jdk = 17
 # list of JDK major versions that are used to run Elasticsearch
-runtime.jdk = 12,11,8
+runtime.jdk = 17,16,15,14,13,12,11,8
```

Comment:
Is there any reason not to add 18 here as well?

Reply:
Since we haven't done that [yet on master](https://github.com/elastic/rally-teams/blob/e60a86959b680d01215f385001d7c1b176cbe4e1/cars/v1/vanilla/config.ini#L21) I thought it could be a separate PR

Reply:
Thanks, that makes sense!